src/test/end-to-end/paths/moveRejectCaseToCaseCloseStateErrorCheck.js

```diff
@@ -0,0 +1,21 @@
+const testConfig = require('./../../config');
+const {createCaseInCcd} = require("../helpers/ccdDataStoreApi");
+const {eventNames} = require('../pages/common/constants.js');
+const assert = require('assert');
+const {rejectCaseEvent, caseDetailsEvent} = require("../helpers/caseHelper");
+let caseNumber;
+
+const verifyState = (eventResponse, state) => {
```

Comment:
Similar to the previous test, i don't believe the verifyState is called?

Reply:
fixed